routes/web.php

```diff
@@ -75,6 +75,20 @@
             });
         });
 
+        Route::prefix(__('route.orders'))->name('orders.')->group(function () {
+            Route::get('/', [ManageOrdersController::class, 'index'])
+                ->name('index');
+
+            Route::get(__('route.order-details') . '/{id}', [ManageOrdersController::class, 'orderDetails'])
```

Comment:
```suggestion
            Route::get('/{id}', [ManageOrdersController::class, 'orderDetails'])
```
Zo heb je ipv: http://127.0.0.1:8000/beheren/bestellingen/bestelling/1 -> http://127.0.0.1:8000/beheren/bestellingen/1

Reply:
Ik had dit eerst ook, alleen kreeg ik errors. Ik kan me niet meer herinneren waarvan, zal kijken of dat nogsteeds zo is.